Scripts/Utilities/UnityBrandColorScheme.cs

```diff
@@ -219,22 +240,87 @@ private static void SetupUnityBrandColors()
+			s_CuratedGradientPairs.Add(new GradientPair(blue, magenta));
+			s_CuratedGradientPairs.Add(new GradientPair(blue, purple));
+			s_CuratedGradientPairs.Add(new GradientPair(magenta, darker));
+			// Slightly lighter pairs added to main curated set
+			s_CuratedGradientPairs.Add(new GradientPair(yellowLight, tealDark));
+			s_CuratedGradientPairs.Add(new GradientPair(orangeDark, orangeLight));
+			s_CuratedGradientPairs.Add(new GradientPair(lime, greenLight));
+			s_CuratedGradientPairs.Add(new GradientPair(red, redLight));
+			s_CuratedGradientPairs.Add(new GradientPair(red, orangeLight));
+			s_CuratedGradientPairs.Add(new GradientPair(lime, orange));
+			s_CuratedGradientPairs.Add(new GradientPair(lime, teal));
+			s_CuratedGradientPairs.Add(new GradientPair(magenta, blueLight));
+			s_CuratedGradientPairs.Add(new GradientPair(blue, blueLight));
+			s_CuratedGradientPairs.Add(new GradientPair(blue, lime));
+			s_CuratedGradientPairs.Add(new GradientPair(blue, yellowLight));
+
+			// Setup dark-luma curated gradient pairs
+			s_CuratedDarkGradientPairs.Add(new GradientPair(tealDark, darkBlue));
+			s_CuratedDarkGradientPairs.Add(new GradientPair(blueDark, darkBlue));
+			s_CuratedDarkGradientPairs.Add(new GradientPair(redDark, darkBlue));
+			s_CuratedDarkGradientPairs.Add(new GradientPair(yellowDark, darkBlue));
+			s_CuratedDarkGradientPairs.Add(new GradientPair(cyanDark, darkBlue));
+			s_CuratedDarkGradientPairs.Add(new GradientPair(greenDark, darkBlue));
+			s_CuratedDarkGradientPairs.Add(new GradientPair(purpleDark, darkBlue));
+			s_CuratedDarkGradientPairs.Add(new GradientPair(darkBlueLight, darkBlue));
+
+			// Setup light-luma curated gradient pairs
+			s_CuratedLightGradientPairs.Add(new GradientPair(redLight, tealDark));
+			s_CuratedLightGradientPairs.Add(new GradientPair(yellowLight, tealDark));
+			s_CuratedLightGradientPairs.Add(new GradientPair(orangeLight,limeDark));
+			s_CuratedLightGradientPairs.Add(new GradientPair(yellowLight, blueDark));
+			s_CuratedLightGradientPairs.Add(new GradientPair(yellowLight, cyanDark));
+			s_CuratedLightGradientPairs.Add(new GradientPair(orangeLight, greenDark));
+			s_CuratedLightGradientPairs.Add(new GradientPair(cyanLight, orange));
+			s_CuratedLightGradientPairs.Add(new GradientPair(yellowLight, purple));
+			s_CuratedLightGradientPairs.Add(new GradientPair(yellowLight, blue));
+			s_CuratedLightGradientPairs.Add(new GradientPair(orangeLight, lime));
+			s_CuratedLightGradientPairs.Add(new GradientPair(redLight, darkBlue));
+			s_CuratedLightGradientPairs.Add(new GradientPair(blueLight, yellow));
+			s_CuratedLightGradientPairs.Add(new GradientPair(blueLight, yellowDark));
+			s_CuratedLightGradientPairs.Add(new GradientPair(greenLight, red));
+			s_CuratedLightGradientPairs.Add(new GradientPair(light, darker));
+			s_CuratedLightGradientPairs.Add(new GradientPair(light, blueLight));
+
+			s_CuratedGradientPairsRange = s_CuratedGradientPairs.Count;
```

Comment:
Addressed in the following commit : https://github.com/Unity-Technologies/EditorVR/commit/72250a914ddf13dea444241564ef7a7d9f29bb68